config/custom_dns_test.go

```diff
@@ -67,7 +69,7 @@ var _ = Describe("CustomDNSConfig", func() {
 		})
 	})
 
-	Describe("UnmarshalYAML", func() {
+	Describe("#CustomDNSEntries UnmarshalYAML", func() {
```

Comment:
Does `#` do anything here?

Reply:
Nope! I will remove it when I work on this next